src/SubMenu.js

```diff
@@ -78,12 +78,14 @@ export default class SubMenu extends AbstractMenu {
                 this.setState({ selectedItem: null });
             });
         } else {
+            this.subMenu.addEventListener('transitionend', () => {
```

Comment:
Yeah you right, I used the new `once` option for `addEventListener`, but it's not compatible with the supported browser list of your library. I will correct this with the traditional variant